docs/product-getting-started-guide/src/main/asciidoc/decision-table-create-proc.adoc

```diff
@@ -0,0 +1,107 @@
++
+
+.Attribute columns window
+image::attrib-column.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* -> *+Create a new Fact Pattern*.
+. Select *Violation* from the *Fact type* pull-down menu, enter `v` in the *Binding* field, and click *+OK*.
++
+
+.Create a new fact pattern window
+image::fact-pattern.png[]
+. Select *Calculation type* -> *Literal value*.
+. Select *Field* and then *type* from the *Field* pull-down menu.
+. Select *Operator* and then *equal to* from the *Operator* pull-down menu.
+. Select *Value options* and enter `Driving while intoxicated,DWI=Driving while under the influence of drugs` in the *Value list (optional)* field.
+. Select *Additional info*, enter `Violation Type` in the *Header (description)* field, and click *Finish*.
++
+
+.Violation Type header
+image::violation-columns.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Calculation type* -> *Predicate* -> *Field* and enter `actualSpeed-speedLimit > $param`.
+. Select *Value options*, then select *Additional info* and enter `Speed Limit (MPH) >` in the *Header (description)* field.
+. Click *Finish*.
++
+
+.Speed Limit (MPH) > header
+image::speed-header.png[]
+. Click *Insert Column*, select *Add a Condition* -> *Pattern* and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Calculation type* -> *Predicate*.
+. Select *Field* and enter `actualSpeed-speedLimit < $param` in the *Field* field.
+. Select *Operator*, select *Value options*, then select *Additional info*.
+. Enter `Speed Limit (MPH) <` in the *Header (description)* field and click *Finish*.
++
+
+.Speed Limit (MPH) < header
+image::speed-less-header.png[]
+
+== Inserting Result columns
+
+. Click *Insert Column*, select *Set the value of a field* -> *Pattern* and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Field* and then *fineAmount* from the *Field* pull-down menu.
+. Select *Value options*, then select *Additional info* and enter `Fine Amount` in the *Header (description)* field.
+. Select the *Update engine with changes* option and click *Finish*.
++
+
+.Fine Amount header
+image::fine-amount.png[]
+. Click *Insert Column*, select *Set the value of a field* -> *Pattern* and select *Violation[v]* from the *Pattern* pull-down menu.
+. Select *Field* and then *points* from the *Field* pull-down menu.
+. Select *Value options*, then select *Additional info* and enter `Points` in the *Header (description)* field.
+. Select the *Update engine with changes* option and click *Finish*.
++
+
+.Point Amount header
+image::point-amount.png[]
+. Click *Columns*, expand *Attribute columns*, and select the *Hide column* option.
++
+
+.Column attributes
+image::hide-column.png[]
+. Click *Editor* -> *Insert* -> *Append row*. Repeat this step to add a total of four table rows.
```

Comment:
If it were me, I would make these last few steps about adding rows a different section, about inserting rows (since this section is after all about inserting results columns, or whatever). And of course :) with an intro statement noting that each row results in a rule, etc. Again, you're welcome to see my GDT doc.

Reply:
I am going to harm u

Reply:
done